### PR TITLE
Fixes #1832 - Separate upgrade guide

### LINKED ISSUE #1832: Improve structure of 'Upgrading' section
At the moment only 0.6.x->0.7.x is complicated enough to require its own 'Upgrading' entry. In later releases, upgrading information is covered in the 'Getting Started' section. 

We should make this clearer, possibly by linking directly into the 'Getting Started' pages from the sidebar for such versions.

### PR DESCRIPTION
**Pull request by kolloch**
from index.md and make it more visible in the side bar.

I also replaced some static site.baseurl references

### CODE REVIEW

File: docs/docs/upgrade/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Upgrading to a Newer Version
+---
+
+# Upgrading to a Newer Version
+
+We generally recommend creating a backup of the ZooKeeper state before upgrading to be able to downgrade in case of problems after an upgrade. This can be done by creating a copy of ZooKeeper's [data directory](http://zookeeper.apache.org/doc/r3.1.2/zookeeperAdmin.html#The+Data+Directory).
+
+## Upgrading a non HA installation
+Upgrading to a newer version of Marathon should be executed in that order:
```
Comment (philipnrmn) [inline docs/docs/upgrade/index.md]: s/that/the following/

File: docs/docs/upgrade/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+1. Tear down the running instance of Marathon.
+1. Install the new version of Marathon.
+1. Start the new version of Marathon and watch the log for a successful start.  
+
+## Upgrading an HA installation
+Upgrading to a newer version of Marathon should be executed in that order:
```
Comment (philipnrmn) [inline docs/docs/upgrade/index.md]: s/that/the following/

**Comment by philipnrmn** [comment]
Nice :)

**Comment by kolloch** [comment]
Ok, thanks, made the suggested changes.

**Pull request merged**